app/src/main/java/org/fossasia/openevent/app/common/data/repository/EventRepository.java

```diff
@@ -94,4 +99,19 @@ public EventRepository(IUtilModel utilModel, IDatabaseRepository databaseReposit
             .build();
     }
 
+    @NonNull
+    @Override
+    public Observable<Event> updateEvent(Event event) {
+        if (!utilModel.isConnected()) {
+            return Observable.error(new Throwable(Constants.NO_NETWORK));
+        }
+
+        event.setTickets(null);
```

Comment:
Yeah. You are right. Should show them in resp time category. How should be they marked as draft? Putting white alpha layer on draft item will do?